packages/docs/content/patterns/utility-actions.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Utility Actions
+intro: Utility actions for a webpage including print, email, RSS and more.
+---
+
+import {
+  DownloadIcon,
+  EmailIcon,
+  LinkIcon,
+  PrintIcon,
+  RssIcon,
+  WhatsNewIcon,
+} from '@cmsgov/design-system';
+
+## Guidance
+
+### Examples
+
+<EmbeddedExample>
+  <div className="ds-l-row">
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <PrintIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        Print
+      </button>
+    </div>
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <EmailIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        Email
+      </button>
+    </div>
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <RssIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        RSS
+      </button>
+    </div>
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <LinkIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        Link
+      </button>
+    </div>
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <DownloadIcon className="ds-u-margin--05" viewBox="0 0 18 18" />
+        Download
+      </button>
+    </div>
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <WhatsNewIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        What's New
+      </button>
+    </div>
+  </div>
+</EmbeddedExample>
+
+### When to use
+
+- If you would like to give the user the option to print or email a page.
+- If you would like to give the user the option to enable RSS.
+- These "utility actions" are considered [ghost buttons](/components/button/#ghost-button) for actions available that do not take priority over a primary or secondary calls to action (e.g., print, email, RSS).
+
+### When to consider alternatives
+
+- If an action (e.g. printing or emailing) is a critical call-to-action on a webpage, consider using a solid or outlined main button.
+- If the utility action is opening a link versus triggering an action, please reference [guidance on links](/foundation/typography/links).
+- Consider carefully which Utility Actions are used on mobile since space is a premium.
+
+### Usage
+
+- This pattern uses a [ghost button](/components/button/#ghost-button) component with a left icon boolean in Figma.
+- Placement should be in page header.
+- Top, right aligned.
```

Comment:
Nit: I think readability could be improved by using full sentences for each bullet point. For example, instead of '- Top, right aligned,' we could say 'The button should be top-right aligned.' Overall, it looks great though.

Reply:
@andream-cmsds what do you think about this?